rocky/rocky/settings_test.py

```diff
@@ -5,3 +5,4 @@
         "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
     },
 }
+COMPRESS_OFFLINE = False
```

Comment:
This should not be changed, all the django-compression stuff should run during build and not be done during requests.